src/owui_ext/shared/notifications.py

```
@@ -0,0 +1,27 @@
+    if not callable(event_emitter):
+        return
+    if not isinstance(content, str) or not content.strip():
+        return
+    try:
+        await event_emitter(
+            {
+                "type": "notification",
+                "data": {"type": level, "content": content.strip()},
+            }
+        )
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)